spec/views/food_index_spec.rb

```diff
@@ -0,0 +1,34 @@
+require 'rails_helper'
+
+RSpec.describe 'Food index', type: :feature do
+  before :each do
+    visit new_user_session_path
+    fill_in 'Email', with: 'tom@gmail.com'
+    fill_in 'Password', with: '123456'
+    click_button 'Log in'
+    visit foods_path
+  end
+
+  describe 'index page for list foods' do
+    it 'I can see the name of food of all other users.' do
+      expect(page).to have_content('Apple')
+    end
+
+    it 'I can see the measurement unit there' do
+      expect(page).to have_content('units')
+    end
+
+    it 'I can see the price there' do
+      expect(page).to have_content('5.0')
+    end
+
+    it 'I can see the button add food' do
+      expect(page).to have_content('Add food')
+    end
+
+    # it 'When I click on add foos, I am redirected to that form add food page.' do
+    #   click_link 'Add food'
+    #   expect(page.current_path).to have_content("foods/#{food.id}")
+    # end
```

Comment:
- [ ] I would recommend you remove commented code to make your tests, look neat.

Reply:
Hi @TedLivist 
Thanks for reviewing it
the tests run correctly but you have to run the migration seeds before it !!!
I write it in the pull request description you can check it please!!